packages/UI/src/context/UserDataContext.js

```diff
@@ -36,7 +36,6 @@ function useUserDataHook() {
       withCredentials: true,
     })
       .then(({ data: currentUser }) => {
-        console.log('Fetching...');
```

Comment:
Believe these console logs were there originally. Can you add them back.

Reply:
Got it thanks